pkg/contracts/src/CVStrategy.sol

```diff
@@ -75,6 +82,13 @@ library StrategyStruct {
         uint256 maxAmount;
     }
 
+    struct ArbitrableConfig {
```

Comment:
We also need TribunalSafe

Reply:
why we would need that there if we have it on the arbitrator?

```
    function registerSafe(address _safe) external {
        arbitrableTribunalSafe[msg.sender] = _safe;
    }
  ```

Reply:
do you think we need to store the tribunal safe on the  cvStrategy if we are already registering on the SafeArbitrator?

Reply:
Not for logic enforcement but maybe more to dispatch to events more easily (Or need an event in RegisterSafe then can be handled in subgraph mappings)